### DISCUSSION TITLE
Custom conversion on specific  columns

### DISCUSSION THREAD

**Discussion by MkJha** [ozlerhakan/poiji#320] [Q&A]
Custom casting on specific columns, while for others default casting should trigger

**Comment by MkJha** [comment]
My use case is to just transform certain column string value to their corresponding actual value for example a cell with value 'President' to be decode as PREZ .  Within custom casting, first calling default casting and on top of that if field type is of specific name , perform custom decoding , would like to know if this is the right approach or we have some other in built feature for this

**Reply by ozlerhakan** [reply]
Hi @MkJha , do you think the same approach would make sense for your use case https://github.com/ozlerhakan/poiji/discussions/287?
